index.js

```diff
@@ -1,18 +1,59 @@
-const leftInput = document.querySelector('#leftInput textarea')
-const rightOutput = document.querySelector('#rightOutput textarea')
+const leftInput = document.querySelector('#leftInput textarea');
+const rightOutput = document.querySelector('#rightOutput textarea');
+const errorIcon = document.getElementById('errorIcon');
 
-leftInput.addEventListener('input', compile)
+leftInput.addEventListener('input', compile);
+
+const localStorageKey = 'json-schema-to-typescript';
+const content = localStorage.getItem(localStorageKey);
+
+if (content) {
+  leftInput.value = content;
+}
+
+const options = {
+  // bannerComment: '',
+  declareExternallyReferenced: true,
+  enableConstEnums: true,
+  // style: { /* prettier config */ },
+  unreachableDefinitions: false,
+  strictIndexSignatures: false
+  // $refOptions: {}
+};
+
+// expose options for advance users
```

Comment:
How would an advanced user know to use this? Maybe we could `console.info` instructions?